docs/05-operations/runbooks/docker-security-baseline.md

```
@@ -1,11 +1,47 @@
+______________________________________________________________________
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)